docs/guide/migrating-to-version-9.md

```diff
@@ -110,6 +122,31 @@ No changes.
 
 ### KInput
 
+#### Structure
+
+* `form-control` class is no longer present
+* `over-char-limit` class is no longer present
+* `has-error` class is no longer present
+* `help` class has been changed to `help-text`
+
+#### Constants, Types & Interfaces
+
+* `SizeArray` and `IconPositionArray` constants are no longer exported
+* `Size` and `IconPosition` types are no longer exported
+
+#### Props
+
+* `overlayLabel` prop is no longer supported
+* `size` prop is no longer supported
+* `iconPosition` prop is no longer supported
+
+#### Slots
+
+* `icon` slot is no longer supported. Instead, you can use new `before` and `after` slots
+
+#### Events
+
+No changes.
```

Comment:
nit: if no changes, this section doesn't need to exist

Reply:
The same order is fine, but if a section isn't needed, there's no reason to clutter up the docs with empty header sections.